src/components/dialogs/BcolErrorDialog.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <v-dialog v-model="bcolDialog" width="45rem" persistent :attach="attach" content-class="bcol-error-dialog">
```

Comment:
When I wrote similar code, I used `:value=` instead of v-model. This may not work here... I'm guessing when you close the dialog, it will try to set bcolDialog (to false), but that property is readonly (getter). I recomment you just use `:value` here. And also test this :)

Reply:
done

Reply:
Fixed.